start_up.py

```diff
@@ -17,9 +17,25 @@ def __init__(self):
                 if i.GetLabel() == 'Nové meranie':
                     i.SetPosition((250, 300))
                     i.Show()
+                    i.Bind(wx.EVT_BUTTON, self.new_click)
                 elif i.GetLabel() == 'Načítať meranie':
                     i.SetPosition((550, 300))
                     i.Show()
+                    i.Bind(wx.EVT_BUTTON, self.load_click)
+
+    def new_click(self, evt):
+        self.Close()
+        self.buttons.new_meas(evt)
+
+    def load_click(self, evt):
+        self.Close()
+        self.buttons.load(evt)

```

Comment:
toto tu je zbytocne kedze o buttony sa uz stara trieda Buttons, zatvaranie okien riesi Handler, je to v branchi implementation